aiohttp/client.py

```diff
@@ -784,9 +784,13 @@ async def _ws_connect(
             extstr = ws_ext_gen(compress=compress)
             real_headers[hdrs.SEC_WEBSOCKET_EXTENSIONS] = extstr
 
-        if not isinstance(ssl, SSL_ALLOWED_TYPES):
+        # We need this code to keep backward compatibility. If ssl is True, we transform it to None
+        if ssl is True:
+            ssl = None
+
+        if not isinstance(ssl, SSL_ALLOWED_TYPES) and ssl is not False:
```

Comment:
Looking at it again, this check happens in 3 different places. So, we'd probably need to refactor the entire check into a function to keep the logic well contained. This is why it hasn't been done before.

Reply:
I reorganized the check method into helper.verify_ssl_type().